### ISSUE TITLE
OpenVLA-SFT Data Collection

### ISSUE THREAD

**Issue by Akila-Ayanthi** [aiming-lab/GRAPE#8]
I'm trying to reproduce OpenVLA-SFT by collecting successful trajectories using Octo-base and Octo-small in simpler-env. My goal is to collect 100 successful trajectories (25 per task). However, even after 24 trials per seed across three different seeds, I still don’t have enough successful rollouts for all four tasks. Specially the task of stacking cubes has 0 successful episodes. 

I noticed that a similar question (Question 3) in [https://github.com/aiming-lab/GRAPE/issues/5](url) and I’d like to understand if there are any recommended solutions or debugging steps for this issue.

Are there any specific hyperparameters or environment settings that could improve the success rate?

Any insights or suggestions would be greatly appreciated!

**Comment by Akila-Ayanthi** [comment]
Hi, I would like to ask if you could provide the fine-tuning trajectories on Simpler

**Comment by zzj1111** [comment]
Maybe I think you need to finetune Base model to get a stronger model in Simpler.
This is the SFT dataset we used to train our OpenVLA-Base:
https://drive.google.com/drive/folders/1HfnZSlYwUwFR_3L3NVRXemkqI3anYZvG
